.github/workflows/pg.yml

```diff
@@ -0,0 +1,42 @@
+name: Build and publish pg lib
```

Comment:
I would prefer the unabbreviated "postgres lib" in the workflow name. It is easier to grasp what's going on then